snap/quota/quota.go

```diff
@@ -71,9 +71,13 @@ type GroupQuotaJournal struct {
 
 	// RateCount/RatePeriod determines the maximum rate of journal writes for
 	// the group. The count is the number of journal messages that can be written
-	// in each period. 0 Values here means there is no limit currently set.
-	RateCount  int           `json:"rate-count,omitempty"`
-	RatePeriod time.Duration `json:"rate-period,omitempty"`
+	// in each period. We treat rate values a bit different as 0 are valid values
+	// here and used to override the journal default rate (which is not unlimited),
+	// and thus the reason we have RateEnabled to tell us whether the values are
+	// to be used.
```

Comment:
Typically each field would have a comment of its own, instead of having a comment in one of them that applies to all. As a bonus, it would work better with editor plugins since at it is they won't show documentation for the bottom two.

Reply:
Really good point, I've changed this